lib/ts/recipe/emailpassword/types.ts

```diff
@@ -27,7 +27,39 @@ const TypeAny = {
     type: "any",
 };
 
+export type TypeInputSetJwtPayloadForSession = (
+    user: User,
+    formFields: TypeFormField[],
+    action: "signin" | "signup"
+) => Promise<{ [key: string]: any }>;
+
+export type TypeInputSetSessionDataForSession = (
+    user: User,
+    formFields: TypeFormField[],
+    action: "signin" | "signup"
+) => Promise<{ [key: string]: any }>;
```

Comment:
Should return `Promise<{ [key: string]: any } | undefined>`

Reply:
done